extensions/px-to-rem-converter/package.json

```
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "px-to-rem-converter",
+  "title": "PX to Rem converter",
+  "description": "Conversions between PX and REM",
+  "icon": "icon.png",
+  "author": "raulanatol",
+  "categories": [
+    "Design Tools",
+    "Developer Tools",
+    "Productivity",
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "px2rem",
+      "subtitle": "PX to REM converter",
```

```suggestion
      "title": "Convert PX to REM",
```

I think only one of those is enough. I struggle a little bit with a proper naming because the conversation is both way. How about something generic?